spec/models/participant_declaration_spec.rb

```diff
@@ -29,6 +32,7 @@
 
       context "for early career teacher was created" do
         let(:profile_type) { :early_career_teacher_profile }
+        let(:course_identifier) { "ecf-induction" }
```

Comment:
Same with `course_identifier` actually

Reply:
> Same with `course_identifier` actually

`course_identifier` is used in the create factory (line 20)